src/store/constants/anime-episodes-types.ts

```diff
@@ -0,0 +1,8 @@
+export const GET_EPISODES_SUCCESS = 'GET_EPISODES_SUCCESS';
+export type GET_EPISODES_SUCCESS = typeof GET_EPISODES_SUCCESS;
+
+export const GET_EPISODES_FAIL = 'GET_EPISODES_FAIL';
+export type GET_EPISODES_FAIL = typeof GET_EPISODES_FAIL;
```

Comment:
it doesn't make sense and that's why: typescript help developers to avoid the mess in the code when they dont know which context which variable should contain. SO:
`const GET_EPISODES: string = "GET_EPISODES";` - makes sense. You always know that it can be only string and any developer cant do a mistake when will use it in an action creator. 
```
export const GET_EPISODES_FAIL = 'GET_EPISODES_FAIL';
export type GET_EPISODES_FAIL = typeof GET_EPISODES_FAIL;
```
- doesn't. Because it literally says "whatever you set in `GET_EPISODES_FAIL` will has correct type". So why do we need typescript at all in this case? 🤷‍♂️